stories/__tests__/SQForm.test.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import {render, screen, waitFor} from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import {BasicForm} from '../SQForm.stories';
+
+window.alert = jest.fn();
+
+afterEach(() => {
+  window.alert.mockClear();
+});
+
+it('renders BasicForm and calls alert on submit', async () => {
+  const {container} = render(<BasicForm />);
+
+  // userEvent.type(screen.getByLabelText(/first name/i), 'First');
+  userEvent.type(container.querySelector('input[name="firstName"]'), 'First');
```

Comment:
Ok let's get them added so we can test these elements with ideal selectors. I have a meeting with the Automated testing team this week to do something similar on SSC. 

Can you create a new issue to insure we add proper identifiers where needed? I think RTL suggests data test id's when possible but it's been awhile since I've looked.

Reply:
Sure. I imagine we'll find more over time as we're writing more tests.

As of right now, they recommend the accessibility-based queries above all because it reflects the experience of the user - here's their current [query priority list](https://testing-library.com/docs/queries/about#priority). Using `data-testid`s would probably be easier and more straightforward, but by relying on them we'd be missing out on testing the true usability of the components, which I feel is pretty important when we're talking about forms.